tests/char_healing_test.cpp

```diff
@@ -334,63 +334,63 @@ TEST_CASE( "healing_rate_medicine with bandages and/or disinfectant", "[heal][ba
     // Bandages heal 1-3 HP per day while awake, 2-6 HP per day while asleep
     SECTION( "bandages only" ) {
         SECTION( "awake" ) {
-            CHECK( bandaged_rate( "head", awake_rest ) == Approx( 1.0f * hp_per_day ) );
-            CHECK( bandaged_rate( "arm_l", awake_rest ) == Approx( 2.0f * hp_per_day ) );
-            CHECK( bandaged_rate( "arm_r", awake_rest ) == Approx( 2.0f * hp_per_day ) );
-            CHECK( bandaged_rate( "leg_l", awake_rest ) == Approx( 2.0f * hp_per_day ) );
-            CHECK( bandaged_rate( "leg_r", awake_rest ) == Approx( 2.0f * hp_per_day ) );
-            CHECK( bandaged_rate( "torso", awake_rest ) == Approx( 3.0f * hp_per_day ) );
+            CHECK( bandaged_rate( "head", awake_rest ) == Approx( 0.86f * hp_per_day ) );
```

Comment:
```suggestion
            CHECK( bandaged_rate( "head", awake_rest ) == Approx( 0.86f * hp_per_day ).epsilon(0.005) );
```
Should give it a tolerance of half a percent of error (which I calculated at least one of these to be under)
I don't think half a percent is an unreasonable amount of error to permit